app/controllers/supervision/supervisions_controller.rb

```diff
@@ -0,0 +1,22 @@
+class Supervision::SupervisionsController < Supervision::Controller
+  def create
+    @supervision.assign_attributes(mission: Mission.find(params[:mission_id]),
+       supervisor: current_user);
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Do not use semicolons to terminate expressions.